src/foam/core/reflow/dashboard/DashboardDAOAgents.js

```diff
@@ -680,13 +684,18 @@ foam.CLASS({
 
   methods: [
     function createSink() {
+      var expr = this.prop;
```

Comment:
Better if we build this support into the view: for 'prop' than having to redo this for groupBy, GridBy, Pivot and a bunch of graph views.